### PR TITLE
test(hc): Set SiloModeTestDecorator.stable to True by default

### PR DESCRIPTION
**Pull request by RyanSkonnord**
Mark all tests that still fail in split-silo mode with an explicit stable=False

### CODE REVIEW

**Comment by RyanSkonnord** [comment]
Requires https://github.com/getsentry/getsentry/pull/12115

Unsure we want to do this while we are still running monolith mode tests in addition to silo-mode tests, because of the added cost of running CI. So this draft also has the purpose of measuring the CI duration.

File: tests/snuba/rules/conditions/test_event_frequency.py [hunk 1]
```diff
@@ -471,7 +471,7 @@ class PerfIssuePlatformIssueUniqueUserFrequencyConditionTestCase(
 
 
 @freeze_time((now() - timedelta(days=2)).replace(hour=12, minute=40, second=0, microsecond=0))
-@region_silo_test
+@region_silo_test(stable=False)
```
Comment (mrduncan) [inline tests/snuba/rules/conditions/test_event_frequency.py:474]: This should be safe to mark `stable=True`, it passed in the most recent test run: https://github.com/getsentry/sentry/actions/runs/6933129416

**Review by mrduncan — APPROVED** [review]
As discussed in office hours - we'll want to remove the duplicate monolith/silo tests at the same time this is merged to prevent extending test durations

**Comment by RyanSkonnord** [comment]
Folding into https://github.com/getsentry/sentry/pull/60291 so as not to increase test load before decreasing it again.

**Pull request closed without merging**